interop/wasi/testdata/args.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+}
+
+//export PrintArgs
+func PrintArgs() {
+	fmt.Printf("os.Args: %v", os.Args)
+}
```

Comment:
naive question (I haven't followed the previous discussion): what's value we could have over directly calling os.Args in main, considering that we end up having longer test codes?

Reply:
In my understanding, it is to make the expected behavior of the wasm module more explicit by calling named exported function rather than calling `_start` and everything is done inside `_start`, which was adrian's concern. It may have only a little difference this time because it has only one output, though. Correct me if I'm wrong @codefromthecrypt 

The discussion is here (https://github.com/tetratelabs/wazero/pull/166#discussion_r796160590) and I remember some are on Slack.

Reply:
ps: I'm also open to having it revert to calling print from `_start`. I think the both style makes sense this time.

Reply:
my point was missed slightly, but almost! I'll try to be clear

1. I think all wasi functions should be tested via export
2. I agree that there's value in testing wasi via tinygo in the adhoc-area

what was missed is that I don't think we do the "re-export for test" approach with tinygo. That's easier and more direct in the main source tree with %.wat even if we temporarily have to check in the result of wat2wasm. Put another way, tests that just call a wasi method re-exported shouldn't rely on tinygo.

Reply:
basically I would expect the interop style test to simply call the stuff via main func (which is invoked via wasi _start).

I would expect proxying the methods to be something easy to do in %.wat or even manually building the opcodes, and that should be with the normal unit tests.

Reply:
Let me summarize your points, firstly points related to this `tests/wasi` file. So, your opinion on this time's change is 
- calling `fmt.Printf("...", os.Args)` in `main` is OK because this is an "interop style test".
- The current style which calls exported `PrintArgs` is the "re-export for test" approach.

Is it right? Then introduction of `PrintArgs` was my misunderstanding, and will revert it. Confirming just in case, because I once read the part below as it discouraged to do that. (sorry for not being able to read the nuances right!

>  I know you insist on using `main` to test this, please comment why, and if we expect args to never be able to be called except from main